data.py-dist

```diff
@@ -61,3 +61,8 @@ DEFAULT_LVMOISCSI_DEVICE_CONFIG = {
 #    'targetIQN': 'target.example',
 #    'SCSIid': 'id'
 }
+
+DEFAULT_FSP_DEVICE_CONFIG = {
+#    'pathtodb': '/root/dir-for-tests',
+#    'pathtoshareddir' : '/root/mount-for-vm0'
```

Comment:
It looks like it doesn't really matter where the files are, so we should probably drop this and let the tests and fixtures just use a temporary directory on the host (`mktemp -d`)... Unless it's possible that future tests import VMs or big VDIs, in which case it would be better to use the `formatted_and_mounted_ext4_disk` fixture to be sure we'll have enough space.

Reply:
I have replaced this with the use of `mktemp -d` for both the db and the shared dir.